judo-runtime-core-jsl-itest/models/NavigationTestModel/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/NavigationTest.java

```diff
@@ -311,7 +340,475 @@ public void testNavigationPersonWithParents() {
+        B1 b2 = b1Dao.create(B1ForCreate.builder().withNumber(2).withA(a2).build());
+
+        Base1 base1 = base1Dao.create(Base1ForCreate.builder()
+                .withNumber(3)
+                .withBs(List.of(b1.adaptTo(B1ForCreate.class), b2.adaptTo(B1ForCreate.class)))
+                .withRelA(a1.adaptTo(A1ForCreate.class))
+                .withCompA(A1ForCreate.builder().withNumber(4).build())
+                .build()
+        );
+
+        Base1 base2 = base1Dao.create(Base1ForCreate.builder()
+                .withNumber(3)
+                .withBs(List.of(b1.adaptTo(B1ForCreate.class), b2.adaptTo(B1ForCreate.class)))
+                .withRelA(a1.adaptTo(A1ForCreate.class))
+                .withCompA(A1ForCreate.builder().withNumber(4).build())
+                .build()
+        );
+
+        assertEquals(7, base1Dao.querySumRelAonBs(base1).orElseThrow());
+
+    }
+
+    @Test
+    @Disabled("https://blackbelt.atlassian.net/browse/JNG-5576")
+    @TestCase("CollectionToObjectNavigationFromAll")
+    @Requirement(reqs = {
+            "REQ-TYPE-001",
+            "REQ-TYPE-005",
+            "REQ-ENT-001",
+            "REQ-ENT-002",
+            "REQ-ENT-004",
+            "REQ-ENT-005",
+            "REQ-ENT-008",
+            "REQ-EXPR-001",
+            "REQ-EXPR-003",
+            "REQ-EXPR-004",
+            "REQ-EXPR-006",
+            "REQ-EXPR-008",
+    })
+    void testCollectionToObjectNavigationFromAll() {
+
+        A1 a1 = a1Dao.create(A1ForCreate.builder().withNumber(1).build());
+        A1 a2 = a1Dao.create(A1ForCreate.builder().withNumber(2).build());
+        A1 a3 = a1Dao.create(A1ForCreate.builder().withNumber(3).build());
+        Base1 base1 = base1Dao.create(Base1ForCreate.builder()
+                .withNumber(3)
+                .withRelA(a1.adaptTo(A1ForCreate.class))
+                .withCompA(A1ForCreate.builder().withNumber(4).build())
+                .build()
+        );
+
+        Base1 base2 = base1Dao.create(Base1ForCreate.builder()
+                .withNumber(3)
+                .withRelA(a2.adaptTo(A1ForCreate.class))
+                .withCompA(A1ForCreate.builder().withNumber(4).build())
+                .build()
+        );
+
+        Base1 base3 = base1Dao.create(Base1ForCreate.builder()
+                .withNumber(3)
+                .withRelA(a3.adaptTo(A1ForCreate.class))
+                .withCompA(A1ForCreate.builder().withNumber(4).build())
+                .build()
+        );
+        assertEquals(6, base1Dao.query().selectList()
+                .stream().map(Base1::getRelA)
+                .map(A1::getNumber)
+                .reduce(0, Integer::sum));
+        // TODO JNG-5576
+        //DerivedAttributeCollector derivedAttributeCollector = derivedAttributeCollectorDao.create(DerivedAttributeCollectorForCreate.builder().build());
+        //assertEquals(6, derivedAttributeCollector.getSumRelAonAllBase1());
```

Comment:
If the test is already disabled, why are the assert disabled as well?

Reply:
[47f9ffe](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/283/commits/47f9ffecc74d58a6d6c9154347db7b61c6c70605)